common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -61,7 +61,7 @@
     % if show_correctness == "never" and (value or status not in ['unsubmitted']):
     <div class="capa_alert">${submitted_message}</div>
     %endif
-    % if msg:
+    % if msg and status.classname != 'unanswered':
```

Comment:
@staubina the span for "message" is included in every problem type HTML file, I believe. Why did you only change these 2?

Also, are we sure we never want to show a message if the problem is unanswered? I feel like I want to look more at what message can be.